lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -98,9 +85,9 @@ def main():
     source_config = Config({
         "LSTEventSource": {
             "max_events" : args.max_events,
+            "default_trigger_type" : 'tib',
```

Comment:
Old run do not have reliable ucts data, so better to use tib if present, as is was done in the previous script

Reply:
Ok. It's fine with me leaving the `'tib'`. But isn't it for newer data the ucts trigger tag more reliable, correcting even possible event jumps? Would it be possible to leave the decision to the event source somehow?

Reply:
unfortunately the event source does not have elements to decide it

Reply:
In any case, I don't understand: aren't flatfields identified by their looks anyway (and the rest of events are pedestals, in dedicated calibration files)

Reply:
The problems are the pedestals, which are wrongly classified as `SUBARRAY` by the UCTS

Reply:
@FrancaCassol @moralejo I agree that the default should be what works best for the current data. The onsite script could make a simple date check when to use tib as default?

But we just have no clear criterion when we should use what and what is more reliable when.

Reply:
> @FrancaCassol @moralejo I agree that the default should be what works best for the current data. The onsite script could make a simple date check when to use tib as default?

+1

> 
> But we just have no clear criterion when we should use what and what is more reliable when. This is also the reason why the source doesn't implement something like that itself

Right, we agreed about that and I do not think we should change it.

Reply:
That does not really make sense, right? Since the point of this script is to be called by osa without any of this kind of logic.

We would need to add a third layer of a script calling a script calling a script.

Is this maybe a task for a (future version) of the run summary? Also write the default trigger type?

Reply:
yes, and one day its should be the task of an online data quality pipeline

Reply:
a subject for another issue #643